omero/developers/Model/EveryObject.txt

```diff
@@ -4,14 +4,6 @@ Glossary of all OMERO Model Objects
 Overview
 --------
 
-In navigating the model objects used by the :doc:`OMERO API
```

Comment:
It's not easy to check out this repository and generate that model documentation which might be annoying. Still, could use includes to make this documentation update easier - that does seem like a good idea.